security/msal/MsalClient.ts

```diff
@@ -58,6 +58,13 @@ export interface MsalClientConfig extends BaseOAuthClientConfig<MsalTokenSpec> {
 
     /** The log level of MSAL. Default is LogLevel.Warning. */
     msalLogLevel?: LogLevel;
+
+    /**
+     * Additional options for the MSAL client ctor. Will be deep merged with defaults, with options
+     * supplied here taking precedence. Use with care, as overriding defaults may have unintended
+     * consequences or fail to work with Hoist's expected usage of the client library.
+     */
+    msalClientOptions?: Partial<msal.Configuration>;
```

Comment:
Am a little concerned about the 'client' containing a 'client', and this sort of puts a fine point on it.  (The internal `client` property had been private.)  Sort of wish we had called our classes "XXXProvider" or (really anything else). 

Not sure what else we would at this point.  Would just calling it `msalOptions`/`auth0Options` help?

Reply:
Let's consider renaming the classes then - because we also have the top-level interfaces hosting these new properties - `MsalClientConfig.msalClientOptions` 🤯 .

If you think that would be manageable at our clients where we've accumulated a few AuthModels already, now is the time to pull the trigger on any better name options.  We DID say it was beta after all! 😄